samples/generated/static-spa/public/app.js

```diff
@@ -241,7 +241,8 @@ function showSigninWidget() {
     });
   
     signIn.showSignInToGetTokens({
-      el: '#signin-widget'
+      el: '#signin-widget',
+      state: JSON.stringify(config.state)
```

Comment:
Is this a temp change to the sample?  If so I'd mention the other Jira in a comment here

Reply:
I mentioned it on the PR, but didn't feel a need to mention it here, since it is totally valid syntax, probably even preferred syntax, to pass state on the call to get tokens, since its directly related to that call. But it **should work** passed as a constructor option. We will get this nailed down on the widget side.